macros/public/store/generate_overview.sql

```diff
@@ -66,15 +66,7 @@
             {{ to_single_json(['data_type', 'is_nullable', 'is_datetime']) }} as {{ re_data.quote_column('data') }}
         from
             columns_casted
-    ) union all
-    (
-        select
-            'dbt_graph' as {{ re_data.quote_column('type') }},
-            null as {{ re_data.quote_column('table_name') }},
-            null as {{ re_data.quote_column('column_name') }},
-            {{- dbt_utils.current_timestamp_in_utc() -}} as {{ re_data.quote_column('computed_on') }},
-            {{ quote_text(dbt_graph)}} as {{ re_data.quote_column('data') }}
-    )order by {{ re_data.quote_column('computed_on')}} desc
+    ) order by {{ re_data.quote_column('computed_on')}} desc
```

Comment:
@mateuszklimek do you think this order by is necessary? `computed_on` is largely going to be the same across the selects.

Reply:
Hm.. why? I mean I believe `computed_on` will be quite different (metrics are compute daily/hours and will have different computed on)

Another thing is that, maybe we actually don't want at as we are sorting it differently on frontend anyway?

Reply:
yeah, since the data isn't fully prepared. we do the sorting in the frontend so it's not really needed